src/Agent.Worker/Release/ReleaseDefinitionToFolderMap.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
+{
+    public sealed class ReleaseDefinitionToFolderMap
+    {
+        [JsonProperty("release_artifactsdirectory")]
+        public string ArtifactsDirectory { get; set; }
```

Comment:
or you can call it ReleaseDirectory, just like we have BuildDirectory for _work/1